src/os_common/os_alloc.c

```diff
@@ -8,31 +8,6 @@
 
 #include "wt_internal.h"
 
-/*
- * On systems with poor default allocators for allocations greater than 16 KB, we provide an option
- * to use TCMalloc explicitly. This is important on Windows which does not have a builtin mechanism
```

Comment:
Just to confirm, since there's not much mention on the ticket. Do we expect the new approach to also work with Windows? I vaguely remember us not using tcmalloc on Windows, so perhaps this removal is just dropping the pretense that it worked :-)

Reply:
We don't test with tcmalloc on Windows. Afaik Google does not support [tcmalloc](https://github.com/google/tcmalloc/blob/master/docs/platforms.md) for windows.

Reply:
Great - thanks!